src_common/ethUstream.c

```diff
@@ -356,36 +356,36 @@ static int processValue(txContext_t *context) {
                  ? context->commandLength
                  : context->currentFieldLength - context->currentFieldPos);
         if (copyTxData(context, context->content->value.value + context->currentFieldPos, copySize)) {
-            return -1;
+            return true;
         }
     }
     if (context->currentFieldPos == context->currentFieldLength) {
         context->content->value.length = context->currentFieldLength;
         context->currentField++;
         context->processingField = false;
     }
-    return 0;
+    return false;
 }
 
 /**
  * @brief Processes the RLP_TO field.
  *
  * @param[in,out] context The transaction context.
  *
- * @return 0 if successful, -1 if an error occurs.
+ * @return false if successful, true if an error occurs.
```

Comment:
I think this is counterintuitive, is there a reason to return false if successful?

Reply:
not really, I found the code like that and kept it this way. Most functions in this app return true when there is an error and false when all is good, I'm not sure why.  Since all those functions have docs, it shouldn't be an issue. What do you think?

Reply:
If it was like this in app-eth then let's keep it that way.